cypress/integration/movie_display.spec.js

```diff
@@ -0,0 +1,60 @@
+describe('Main Display', () => {
+    const baseUrl = 'http://localhost:3000';
+    it('should be able to visit the base url and see a grid display of movie posters', () => {
+        cy.visit(baseUrl);
+    });
+    beforeEach(() => {
+        cy.visit(baseUrl);
+    });
+    it('should display a message while the posters are loading to inform the user', () => {
+        cy
+        .get('h2')
+        .contains('The movies are on their way!'
+        )
+    });
+
+    it('should display an error if movies are not loading', () => {
```

Comment:
Let's move this to the movie_info.spec.js file for testing there. Glad we figured this one out though!